Print_A_Bot/djwifi/templates/djwifi/configure.html

```diff
@@ -0,0 +1,246 @@
+{% extends 'djwifi/base.html' %}
```

Comment:
this page and the associated javascript generates this page and then calls django via jQuery and javascript / json

doesn't use any of the newer frameworks but it works well